src/app/frontend/common/components/actionbar/detailactions/exec/template.html

```diff
@@ -14,11 +14,11 @@
 limitations under the License.
 -->
 
-<button mat-icon-button
-        color="accent"
-        class="kd-toolbar-action"
-        matTooltip="Exec into pod"
-        i18n-matTooltip
-        (click)="onClick()">
+<a mat-icon-button
+   color="accent"
+   class="kd-toolbar-action"
+   matTooltip="Exec into pod"
+   i18n-matTooltip
+   [href]="getHref()">
```

Comment:
Previously the logs view was opened in a new window. Do you think it is better to make this a tag target="_blank?"

Reply:
It was done on purpose. We will open in the same window and later on there will be a swich in the settings allowing to choose what do you prefer.